lib/matplotlib/artist.py

```diff
@@ -430,6 +430,8 @@ def contains(artist: Artist, event: MouseEvent) -> bool, dict
               implementation of the respective artist, but may provide
               additional information.
         """
+        if not callable(picker):
```

Comment:
I think it‘s reasonable to keep this check. And not just fail when trying to call _contains